packages/web3-eth/src/index.js

```diff
@@ -382,9 +377,6 @@ var Eth = function Eth() {
     // add ABI
     this.abi = abi;
 
-    // add ENS
-    this.ens = new ENS(this);
-
```

Comment:
This will introduce a [breaking change](https://web3js.readthedocs.io/en/v1.5.0/web3-eth-ens.html) in `1.x`. I am not sure if we are intended to make such changes while maintaining `1.x`. @jdevcs @spacesailor24 @luu-alex Share your thoughts.

Reply:
Removing ENS  from the web3 package might be a little out of scope for maintaining `1.x`. This issue is fixed in `4.x` and might be too big of a change in `1.x`.